protocol/avdl/crypto.avdl

```diff
@@ -45,6 +45,7 @@ protocol crypto {
     KID kid;
     EncryptedBytes32 ciphertext;
     BoxNonce nonce;
+    BoxPublicKey ePublicKey;
```

Comment:
Can we just call it publicKey?  The 'e' doesn't mean much to me, especially three weeks from now.